Casks/m/morpheus.rb

```diff
@@ -8,10 +8,7 @@
   desc "Modelling environment for multi-cellular systems biology"
   homepage "https://morpheus.gitlab.io/"
 
-  livecheck do
-    url "https://imc.zih.tu-dresden.de/morpheus/packages/mac/"
-    regex(/href=.*?Morpheus[._-](\d+(?:\.\d+)*)\.dmg/i)
-  end
+  deprecate! date: "2024-04-15", because: :no_longer_available
```

Comment:
```suggestion
  deprecate! date: "2024-04-15", because: :discontinued
```

I think discontinued makes more sense here. We usually only use `no_longer_available` when it can not longer be downloaded.